dotCMS/src/main/java/com/dotcms/content/elasticsearch/business/ESContentletAPIImpl.java

```diff
@@ -1810,7 +1810,12 @@ private boolean deleteContentlets(List<Contentlet> contentlets, User user,
 
     private void sendDeleteEvent (final Contentlet contentlet) throws DotHibernateException {
 
-        HibernateUtil.addAsyncCommitListener(() -> this.contentletSystemEventUtil.pushDeleteEvent(contentlet), 1000);
+        HibernateUtil.addAsyncCommitListener(
+                () -> {
+
+                    this.isInodeIndexed(contentlet.getInode());
```

Comment:
If this event is fired after a delete operation what is the **isInodeIndexed** method validating?

Reply:
agree going to remove